src/main/handlers/Utilities/Utilities.class.ts

```diff
@@ -115,7 +115,7 @@ class Utilities {
       }
 
       if (
-        (prerequisite.id === 'wsl' && stdOutput) ||
+        (prerequisite.id === 'wsl' && stdOutput) || (prerequisite.id === 'ps1ExecutionPolicy' && stdOutput.includes('RemoteSigned')) ||
```

Comment:
Run `npm run format` before pushing the code just to ensure formatting is correct and github actions are not failing.